Safer checking for REGISTER_SUPER_ACK and UNREGISTER_SUPER

randomize00 opened the pull request:
safer checking for REGISTER_SUPER_ACK and REGISTER_SUPER_NAK ( only those packet from supernode in federation are legal )

For more details please see #603

randomize00 committed:
Update sn_utils.c

safer checking for REGISTER_SUPER_ACK and REGISTER_SUPER_NAK ( only those packet from supernode in federation are legal )

Logan007 commented:
While the REGISTER_SUPER_ACK is not sent out by edges, the UNREGISTER_SUPER, however, is supposed to be sent only by edges. So, we cannot apply the same condition to UNREGISTER_SUPER if I am not mistaken here.

Would you please correct that in the code – and do a`add`, `commit` and `push` (to the same branch, it automatically gets into this pull request)?

randomize00 committed:
Update sn_utils.c

fix the checking for UNREGISTER_SUPER ( the logic is opposite to for REGISTER_SUPER_ACK )

randomize00 commented:
OK. Fixed.

Logan007 committed:
updated sn_utils.c

typo

Logan007 commented:
Thank you!
